src/files/header/BreathingCloud.css

```diff
@@ -0,0 +1,21 @@
+.BreathingCloud {
```

Comment:
I've just realized that having two different animations for indicating the same thing using the same icon  looks a bit weird.

Can we replace this with the more subtle one  from the File list? (without drop shadow)

> ![2022-08-09_22-34](https://user-images.githubusercontent.com/157609/183755772-7d05e8db-5728-46a1-8863-f4f7b9aa1a0f.png)